telegram/ext/callbackqueryhandler.py

```diff
@@ -76,9 +76,13 @@ class CallbackQueryHandler(Handler):
             :class:`telegram.ext.JobQueue` instance created by the :class:`telegram.ext.Updater`
             which can be used to schedule new jobs. Default is ``False``.
             DEPRECATED: Please switch to context based callbacks.
-        pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not ``None``, ``re.match``
-            is used on :attr:`telegram.CallbackQuery.data` to determine if an update should be
-            handled by this handler.
+        pattern (:obj:`str` | `Pattern` | :obj:`callable`, optional): Regex pattern. If not
```

Comment:
Actually, allowing a type here could also be good, making `CQH` act similar to `TypeHandler` when we have custom types as callback data.